greengrass_parsec_workshop/build_demo.sh

```diff
@@ -28,12 +30,14 @@ function dirty_build_on_new_comits() {
 }
 
 function build_greengrass_patched() {
-pushd ./aws-greengrass-parsec-provider/parsec-greengrass-run-config/docker/
-docker build . \
-       --build-arg BUILD_TS=${DIRTY_TS} \
-       --tag parallaxsecond/greengrass_patched:latest \
-       --progress plain
-popd
+  echo "Build greengrass patched ..."
+  pushd ./aws-greengrass-parsec-provider/parsec-greengrass-run-config/docker/
+  docker build . \
+         --build-arg BUILD_TS=${DIRTY_TS} \
```

Comment:
Why do we require/use this parameter? In my testing I've removed it and the `dirty_build_on_new_commits` function I haven't noticed any issues.